mail/rspamd/src/opnsense/mvc/app/models/OPNsense/Rspamd/RSpamd.xml

```diff
@@ -1,6 +1,7 @@
 <model>
   <mount>//OPNsense/Rspamd</mount>
   <description>rspamd anti spam filter</description>
+  <version>0.0.0</version>
```

Comment:
i think this translates to "unversioned"

Reply:
0.0.0 is the default.

Reply:
yes, that's true. 0.0.0 is unversioned, which is the default.

Reply:
yes - and no bigger version exists as there is nothing that needs a migration.

Reply:
It's all the same, we don't need this line if we don't use it.

Reply:
I always thought 1.0.0 is initial stable?

Reply:
no, the default is 0.0.0 and any different version will trigger the migration update.

The model version is not visible to the user (only the PLUGIN_VERSION)

Reply:
guys, it's pretty simple: either set this to something not 0.0.0 or remove the change until a migration is needed.

Reply:
@fichtner than I'll keep it out.